llvm/lib/CodeGen/AsmPrinter/DwarfDebug.h

```diff
@@ -925,6 +926,7 @@ class DwarfDebug : public DebugHandlerBase {
 
   /// Sets the current DWARF5AccelTable to use.
   void setCurrentDWARF5AccelTable(const DWARF5AccelTableKind Kind) {
+    CurrentKind = Kind;
```

Comment:
Could skip this variable and check `CurrentDebugNames == &AccelDebugNames` or similar.